src/Postmark/Model/PostmarkInboundMessage.cs

```diff
@@ -152,7 +152,7 @@ public class Attachment
         public string Name { get; set; }
         public string Content { get; set; }
         public string ContentType { get; set; }
-        public string ContentLength { get; set; }
+        public int ContentLength { get; set; }

```

Comment:
This should be typed as `long`.
```suggestion
        public long ContentLength { get; set; }

```